CHANGELOG.md

```diff
@@ -12,7 +12,7 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
-## v1.3.0 (WIP)
+## v1.3.0 (2021-06-14)
```

Comment:
Suggest just setting the current date. If this PR takes more than 1 day to be reviewed, then you can just update this date without re-requesting our reviews

```suggestion
## v1.3.0 (2021-06-10)
```